docs/aws-eks-tutorial.md

```diff
@@ -0,0 +1,472 @@
+
+chmod +x kubectl
+ln -s $(pwd)/kubectl /usr/local/bin/kubectl
+```
+
+```
+# For linux
+curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl
+
+chmod +x kubectl
+ln -s $(pwd)/kubectl /usr/local/bin/kubectl
+```
+
+### Download the aws-iam-authenticator
+
+A tool to use AWS IAM credentials to authenticate to a Kubernetes cluster.
+
+```sh
+# For macOS
+wget -O heptio-authenticator-aws https://github.com/kubernetes-sigs/aws-iam-authenticator/releases/download/v0.3.0/heptio-authenticator-aws_0.3.0_darwin_amd64
+
+chmod +x heptio-authenticator-aws
+ln -s $(pwd)/heptio-authenticator-aws /usr/local/bin/heptio-authenticator-aws
+```
+
+```sh
+# For linux
+wget -O heptio-authenticator-aws https://github.com/kubernetes-sigs/aws-iam-authenticator/releases/download/v0.3.0/heptio-authenticator-aws_0.3.0_linux_amd64
+
+chmod +x heptio-authenticator-aws
+ln -s $(pwd)/heptio-authenticator-aws /usr/local/bin/heptio-authenticator-aws
+```
+
+### Modify providers.tf
+
+Choose your region. EKS is not available in every region, use the Region Table to check whether your region is supported: https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services/
+
+Make changes in providers.tf accordingly (region, optionally profile)
+
+### Terraform apply
+
+Start AWS EKS service and create a fully operational Kubernetes environment on your
+AWS cloud.
+
+```
+terraform init
+terraform apply  # This step may take more than 10 minutes to finish. Just wait patiently.
+```
+
+### Configure kubectl
+
+Export the configuration of the EKS cluster to `.config`. This will allow the `kubectl` command line client to access our newly deployed cluster.
+
+```
+mkdir .config
+
+terraform output kubeconfig > .config/ekskubeconfig 
+# Save output in ~/.kube/config and then use the following env prarm
+
+export  KUBECONFIG=$KUBECONFIG:$(pwd)/.config/ekskubeconfig
+# Remember to run the above command again with the right folders when you open a new shell window later for TiDB related works.
+```
+
+### Configure config-map-auth-aws
+
+Setup a ConfigMap on Kubernetes so the node instances can have proper IAM roles.
+
+```
+terraform output config-map-aws-auth > .config/config-map-aws-auth.yaml
+# save output in config-map-aws-auth.yaml
+
+kubectl apply -f .config/config-map-aws-auth.yaml
+```
+
+### See service and nodes coming up
+
+Your `kubectl` command line client should now be installed and configured. Check the installed services and kubernetes nodes:
+
+```
+kubectl get svc
+
+kubectl get nodes
+```
+
+More info about getting started with Amazon EKS[here](https://docs.aws.amazon.com/eks/latest/userguide/getting-started.html)
+
+### Deploy Kubernetes Dashboard
+
+#### Step 1: Deploy the Dashboard
+
```

Comment:
Maybe add (optional) here too.  If they do deploy on a bastian host, it will be harder to get the dashboard to work.